src/robot_properties_solo/resources/xacro/leg.xacro

```diff
@@ -58,27 +58,27 @@
                 <xacro:if value="${is_front}">
                     <origin xyz="0.01950 0 0" rpy="0 0 0" />
                     <geometry>
-                        <mesh filename="package://robot_properties_solo/meshes/${mesh_ext}/solo12/solo12_hip_fe_fl.${mesh_ext}"/>
+                        <mesh filename="package://meshes/${mesh_ext}/solo12/solo12_hip_fe_fl.${mesh_ext}"/>
```

Comment:
Doesn't omitting the package name break the ability to visualise in RViz and other viewers? For PyBullet, you can add additional search paths so it works with standard URDFs out of the box

Reply:
Yes this most likely break it... I thought I made sure the urdf was able to load in both RVIZ+GepttoViewer+PyBullet
I would like to preserve this feature please.

Reply:
I see! Thanks for pointing it out.